src/components/AddLiquidityModal.js

```diff
@@ -110,10 +110,10 @@ export default function CurrencySelectionModal(props) {
             }
           </Col>
           <Col className="align-self-center">
-            {convertToE24Base5Dec(inputs.state.addLiquidityModal.tokenPerNear)}
+            {convertToDecimals(inputs.state.addLiquidityModal.tokenPerNear, 24)}
```

Comment:
this is not correct. The token decimals should be taken from the metadata. If decimals are not provided then we need:
* don't format with decimals
* add a warning icon (eg exclamation mark)
* on hovering on the icon, explain that we can't properly format he amount because we don't have a decimal information.

Reply:
This value is coming from contract.
assumed 24 decimals values from contract.
How are we going to handle two tokens with decimals in contract ?
i can update it accordingly.

Reply:
understood, adding decimals attribute here, and will use it.